stories/topics/con-aws-update-backup-data-file.adoc

```diff
@@ -25,8 +25,9 @@ Please ensure the referenced backup vault exists and the AWS Backup IAM role pro
 * `aws_region` is the region where the foundation deployment is deployed.
 * `aws_s3_bucket` is the name of the S3 bucket where the backup files are stored. 
 This creates a new bucket if one does not already exist. 
-Every backup is stored in the bucket. 
-For guidance on bucket naming, see link:https://www.google.com/url?q=https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html&sa=D&source=docs&ust=1682584492105680&usg=AOvVaw0WgUwkieoKwP4tRrvSpFBC[Bucket naming rules]. For more information on creating S3 buckets, see link:https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html[AWS Creating A Bucket] in the AWS documentation.
+Every backup is stored in the bucket. +
+For guidance on bucket naming, see link:https://www.google.com/url?q=https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html&sa=D&source=docs&ust=1682584492105680&usg=AOvVaw0WgUwkieoKwP4tRrvSpFBC[Bucket naming rules]. +
+For guidance on creating S3 buckets, see link:https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html[AWS Creating a bucket].
```

Comment:
put a blank line between these two xrefs

Reply:
I added the `+` to create a line break. If I add a blank line and remove the +, things look off again.

It looks like this when removing the + and adding a blank line
<img width="735" alt="Screenshot 2023-07-27 at 2 53 09 PM" src="https://github.com/ansible/aap-docs/assets/112966709/89e79df6-eac0-45e2-851f-eb5647183f6f">



I wanted it to look like below
<img width="694" alt="Screenshot 2023-07-27 at 2 49 28 PM" src="https://github.com/ansible/aap-docs/assets/112966709/b9def54e-4c74-4bd8-a412-b441494277d7">
ebf-bb4786ad8bd2">